src/main/kotlin/fr/social/gouv/agora/usecase/reponseConsultation/InsertReponseConsultationUseCase.kt

```diff
@@ -13,20 +15,26 @@ import org.springframework.stereotype.Service
 
 @Service
 class InsertReponseConsultationUseCase(
+    private val contentSanitizer: ContentSanitizer,
     private val consultationPreviewAnsweredRepository: ConsultationPreviewAnsweredRepository,
     private val insertReponseConsultationRepository: InsertReponseConsultationRepository,
     private val consultationResponseRepository: GetConsultationResponseRepository,
     private val questionRepository: QuestionRepository,
     private val insertConsultationResponseParametersMapper: InsertConsultationResponseParametersMapper,
     private val consultationPreviewPageRepository: ConsultationPreviewPageRepository,
 ) {
+    companion object {
+        private const val AUTRE_QUESTION_MAX_LENGTH = 200
+        private const val OPEN_QUESTION_MAX_LENGTH = 400
+    }
 
     fun insertReponseConsultation(
         consultationId: String,
         userId: String,
         consultationResponses: List<ReponseConsultationInserting>,
     ): InsertResult {
         if (consultationResponseRepository.hasAnsweredConsultation(consultationId = consultationId, userId = userId)) {
+            println("⚠️ User already answer the consultation")
```

Comment:
Plutôt le format suivant pour qu'on ait une meilleure idée de quelle fonctionnalité est en erreur:
`⚠️ Insert response consultation error: user has already answered this consultation`

Reply:
done